docs/misc/message_system.md

```diff
@@ -125,10 +125,12 @@ Structure of config, types and optionality of specific keys can be found in the
                         "ios": "13",
                         "chromeos": "*"
                     },
+                    // revision is optional
```

Comment:
I mean all the items are optional nope?

Reply:
Not really. If you want to include all versions, you use `*`, if none `!`.

I can make `revision` to work the same. `*` all revisions, "blablabla" if you want specific.

Reply:
I am not sure to be honest, so I think I will leave it up to you. I was even considering if the desktop/mobile/web fields should not accept the revision directly instead of version? 🤔  Because version is a subset of revision basically.

Reply:
I mean that you could supply 22.2.1 or bcdbcdb..

Reply:
Discussed on Slack, this will do 👌 .